api/api_sources/resources/csv/SpecificUseCode.csv

```diff
@@ -21,3 +21,4 @@ MI,Mine site
 MT,Mine tailings
 BS,Burn scar
 IS,Industrial site
+NA,None
```

Comment:
Currently,  we are running the migration progressively, so make sure new values should be migrated as fresh new migration.